doc/plugins/unittest.rst

```diff
@@ -0,0 +1,69 @@
+###############
+Spyder Unittest
+###############
+
+**Spyder-unittest** is a plugin that integrates popular unit test frameworks with Spyder, allowing you to run test suites and view the results in the IDE.
+
+.. image:: /images/unittest/unittest-run.png
+   :alt: Spyder Unittest in Spyder
+
+Using unittest inside Spyder allows you to take advantage of their web interface alongside Spyder’s powerful features such as the Variable explorer, console and debugger.
+
+===================
+Installing Unittest
+===================
+
+.. code-block:: bash
+
+   conda install -c spyder-ide spyder-unittest
+
+.. important::
+
+   Please make sure you have Python's pytest package installed if you are going to use that program as well.
+
+.. code-block:: bash
+
+   conda install pytest
+
+Restart Spyder in order to be able to use the plugin.
+
+==============
+Using Unittest
+==============
+
+When Unittest is installed, it will be available under the menu item :menuselection:`View --> Panes --> Unit testing`.
```

Comment:
```suggestion
When Spyder-unittest is installed, it will be available under the menu item :menuselection:`View --> Panes --> Unit testing`.
```

Unittest is the name of standard library module for testing, so I think it's better to be more explicit here.

Reply:
changed.